osversion/platform_compat_windows_test.go

```diff
@@ -1,68 +1,86 @@
 package osversion
 
 import (
+	"fmt"
 	"testing"
 )
 
-// Test the platform compatibility of the different
-// OS Versions considering two ltsc container image
-// versions (ltsc2019, ltsc2022)
+// Test the platform compatibility of the different OS Versions
 func Test_PlatformCompat(t *testing.T) {
-	for testName, tc := range map[string]struct {
-		hostOs    uint16
-		ctrOs     uint16
+	for _, tc := range []struct {
+		hostOS    uint16
+		ctrOS     uint16
 		shouldRun bool
 	}{
-		"RS5Host_ltsc2019": {
-			hostOs:    RS5,
-			ctrOs:     RS5,
+		{
+			hostOS:    LTSC2019,
```

Comment:
RS5 and LTSC2019 are the same too - is the intent to refer to OSVersions in terms of ltsc* when possible?

Reply:
Yes, I think the LTSC version naming is clearer.